.changelog/_1330.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:bug
+acl: **(Enterprise only)** fix namespace policy evaluation in the context of partitions
```

Comment:
Might be worth clarifying that it was specifically for namespace_prefix with the empty string (if I understood the bug correctly).

Reply:
It was slightly more general in that if you had an exact and a prefix match that were factors in the same authz request, and only the prefix rule set `policy = <anything>` then it would be discarded